index.html

```diff
@@ -1,32 +1,164 @@
+              Stories</h3>
+                 <ul class="frame-container">
+                   <li class="frame client">CANOPY</li>
+                   <li class="frame Counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame role">Back End Dev</li>
+                   <li class="frame counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame year">2015</li>
+                 </ul>
+                 <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <ul class="language-container">
+                   <li class="language html">HTML</li>
+                   <li class="language css">CSS</li>
+                   <li class="language java">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+    </section>
+    <section class="contact-container">
+        <div class="one">
+         <h3 class="about">About Myself</h3>
+         <p class="bio">Hello I’m a software developer!
+             I can help you build a product , feature or website. 
+             Look through some of my work and experience! 
+             If you like what you see and have a project you need coded, don’t hesitate 
+             to contact me.
+         </p>
+         <h4 class="connect"><a class="connect-bottom" href="#">LET’S CONNECT</a></h4>
+           <ul class="icon bicon">
+             <li class="github"><a href="#"><img src="/img/github.svg" alt="github"></a></li>   
+             <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt="linkedin"></a></li>
+             <li class="v"><a href="#"><img src="/img/v.svg" alt="v"></a></li>
+             <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt="twiter"></a></li>
+             <li class="m"><a href="#"><img src="/img/m.svg" alt="m"></a></li>
+             
+             
+         </ul>
+         <button class="card-btn resume">Get my resume</button>
+        </div>
+        <div class="two">
+            <div class="title-language">
+                <h2 class="tl">Language</h2>
+                <img src="/img/Icon - Chevron - Down - Larger.svg" alt="" class="dropdow">
+            </div>
+            <ul class="fotter-java">
+              <li class="space"><img src="/img/java.svg" alt="Javascript" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">Javascript</h3></li>
+            </ul>
+            <ul class="fotter-html">
+              <li class="space"><img src="/img/html.svg" alt="html" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">HTML</h3></li>
+            </ul>
+            <ul class="fotter-css">
+              <li class="space"><img src="/img/css.svg" alt="css" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">CSS</h3></li>
+            </ul>
```

Comment:
- [ ] As per requested by the previous reviewer, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") is also a list **ul** and **each of them is a li**
As of now, you set each of them as **ul**, instead you should wrap them in one **ul** and each of them should be a **li** 🫐 🍌 

- [ ] Your list of technologies should be centered on the design from Figma. It is good to follow the provided design because they are designed to look professional 📱 ☄️ 

Your design 👇🏽 

![Screenshot from 2023-07-08 23-15-05](https://github.com/ali-techworld/Portfolio-Setup/assets/10439283/d6547c6b-4357-4bfc-9a0e-f94ac78fd174)

Design from Figma 👇🏽 

![Screenshot from 2023-07-08 23-24-28](https://github.com/ali-techworld/Portfolio-Setup/assets/10439283/84f32469-3642-4902-9395-ad54f47c8148)